indico/core/storage/models.py

```diff
@@ -136,6 +136,19 @@ def size(cls):
         )
 
     @declared_attr
+    def md5(cls):
+        """
+        An MD5 hash of the file.
+
+        Automatically assigned when `save()` is called.
+        """
+        return db.Column(
+            db.String(32),
```

Comment:
I actually stopped to think about it and was wondering if it makes any sense given that it's always 32 char long.